docs/source/specifications/geodetictiffgrids.rst

```diff
@@ -636,7 +636,7 @@ It contains 5 subgrids. All essential metadata to list the subgrids and their
 georeferencing is contained within the first 3 KB of the file.
 
 The file size is 4.8 MB using DEFLATE compression and floating-point predictor.
-To be compared with the 83 MB of the original .gsb file.
+To be compared with the 83 MB of the original .gsb file that is lossless with it.
```

Comment:
I'm not sure this formulation is very clear. I'd rather change the previous line to "The file size is 4.8 MB using DEFLATE compression and floating-point predictor, which are lossless compression parameters"

Reply:
> which are lossless compression parameters"

No, what i meant is that it is losssless with original gsb file, not that the float compressed TIFF is lossless. That is obvious.